### PR TITLE
Break long word in user message

### PR DESCRIPTION
**Pull request by TP-O**
Before:
image

After:
image

### CODE REVIEW

File: src/lib/components/chat/ChatMessage.svelte [hunk 1]
```diff
@@ -116,7 +116,7 @@
 {#if message.from === "user"}
 	<div class="group relative flex items-start justify-start gap-4 max-sm:text-sm">
 		<div class="mt-5 h-3 w-3 flex-none rounded-full" />
-		<div class="whitespace-break-spaces rounded-2xl px-5 py-3.5 text-gray-500 dark:text-gray-400">
+		<div class="whitespace-break-spaces break-all rounded-2xl px-5 py-3.5 text-gray-500 dark:text-gray-400">
```
Comment (coyotte508) [inline src/lib/components/chat/ChatMessage.svelte:119]:
```suggestion
		<div class="whitespace-break-spaces break-words rounded-2xl px-5 py-3.5 text-gray-500 dark:text-gray-400">
```

Reference:

- https://tailwindcss.com/docs/word-break#break-words
- https://tailwindcss.com/docs/word-break#break-all

File: src/lib/components/chat/ChatMessage.svelte [hunk 2]
```diff
@@ -116,7 +116,9 @@
 {#if message.from === "user"}
 	<div class="group relative flex items-start justify-start gap-4 max-sm:text-sm">
 		<div class="mt-5 h-3 w-3 flex-none rounded-full" />
-		<div class="whitespace-break-spaces rounded-2xl px-5 py-3.5 text-gray-500 dark:text-gray-400">
+		<div
+			class="w-full whitespace-break-spaces break-words rounded-2xl px-5 py-3.5 text-gray-500 dark:text-gray-400"
```
Comment (coyotte508) [inline src/lib/components/chat/ChatMessage.svelte:120]:
```suggestion
			class="max-w-full whitespace-break-spaces break-words rounded-2xl px-5 py-3.5 text-gray-500 dark:text-gray-400"
```

**Review by coyotte508 — APPROVED** [review]
Thanks @TP-O :)

**Comment by gary149** [comment]
It would be interesting to do it for assistant messages also?

image

**Pull request merged**